README.md

```diff
@@ -37,7 +37,7 @@ before_install:
   # Download the script
   - curl "${GRAVIS}.install-jdk-travis.sh" --output .install-jdk-travis.sh
```

Comment:
IMO it is better to update this to use `source ~/.install-jdk-travis.sh` , so that gravis isnt polluting the source tree